scripts/downloadAndExtractEdge.js

```diff
@@ -36,7 +37,7 @@ function fetchDownloadUrl(version) {
   }
   console.log(`Downloading Microsoft Edge DevTools version ${fullVersion}`);
   for (let object of jsonObjects) {
-    if (object.product === 'Microsoft Edge DevTools' && object.release === fullVersion && object.platform === 'Windows x64') {

```

Comment:
Why was this check removed?

Reply:
We stopped producing different source file downloads for different platforms since it's all the same.  For this latest version, I left the platform blank, so we need to remove this check.